SoftwareCo/SoftwareCo/Utils/Constants.cs

```diff
@@ -5,8 +5,8 @@ internal static class Constants
         internal const string EditorName = "visualstudio";
         internal const long DEFAULT_SESSION_THRESHOLD_SECONDS = 60 * 15;
 
-        internal const string api_endpoint = "https://api.software.com";
-        internal const string url_endpoint = "https://app.software.com";
+        internal const string api_endpoint = "https://stagingapi.software.com"; // "https://api.software.com";
```

Comment:
Remember to change this back before merging

Reply:
right, thanks